crowbar_framework/lib/version_compare.rb

```diff
@@ -1,4 +1,4 @@
-class PlatformVersion
+class VersionCompare
```

Comment:
I'd prefer to avoid the renaming for Cloud 6. It's unclear if this is used anywhere else and so on, and I don't want to risk a regression for this.